connectors/src/connectors/notion/temporal/worker.ts

```diff
@@ -15,6 +15,7 @@ export async function runNotionWorker() {
     activities,
     taskQueue: QUEUE_NAME,
     connection,
+    reuseV8Context: true,
```

Comment:
Can we do Notion only at first to monitor the impact? You can monitor from [here](https://app.datadoghq.eu/notebook/124866/investigate-connector-workers).